### ISSUE TITLE
Consistent ordering between unique_edge_map and cotmatrix_entries

### ISSUE THREAD

**Issue by Sentient07** [libigl/libigl-python-bindings#216]
Hello,

I think it would be really useful to have unique_edge_map and cotmatrix_entries ordered in the same manner. I agree that the explanation provided in the documentation is accurate, but I totally missed this small detail and assumed that they're in 1-1 ordering. I think it would be immensely useful to have consistent ordering here. Thanks a lot for this maintaining this library and your efforts, much appreciated!

Cheers, 
Ramana

**Comment by alecjacobson** [comment]
I agree it would be really useful. Unfortunately it seems to problematic to change now.
